packages/server/test/unit/config_spec.js

```diff
@@ -1146,53 +1146,53 @@ describe('lib/config', () => {
         return config.mergeDefaults(obj, options)
```

Comment:
Only the order of the options changed for these tests, since everything is now alphabetical. The existence of the keys and their values remain the same.